src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig

```diff
@@ -30,6 +30,14 @@
     <div class="sf-tabs">
         <div class="tab">
             {% set exception_as_array = exception.toarray %}
+            {% set _exceptions_with_user_code = [] %}
+            {% for i, e in exception_as_array %}
+                {% for trace in e.trace %}
+                    {% if (trace.file is not empty) and ('/vendor/' not in trace.file) and not loop.last %}
```

Comment:
That info is not easily available right now. It might be a good idea for 4.0 to move some of this 'business logic' to `ExceptionController` instead so overridden templates can also profit from it.

Pushed the change adding `/var/cache/` for now.